tools/webinstall/install.sh

```diff
@@ -1153,6 +1160,85 @@ install_linux_manual() {
     _CLEANUP_VERSION=""
 }
 
+# check_environment checks the environment for the kraftkit group.
+# If the group already exists, it means that it owns all default files.
+# $1: the group name
+# Returns:
+# Code: 0 if group doesn't exist, 1 if it exists
+check_environment() {
```

Comment:
I'd rename `environment` (in the two newly introduced methods) to `usergroup` to prevent confusion with the idea of environmental variables.